app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncMessage.java

```diff
@@ -0,0 +1,142 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sender_sync_message")
+public class SenderSyncMessage extends AbstractEntity {
+	
+	private static final long serialVersionUID = 1L;
+	
+	// TODO add ACKNOWLEDGED status, which will be set after successful processing of the receiver response
+	public enum SenderSyncMessageStatus {
+		NEW,
+		SENT,
+		ERROR
+	}
+	
+	@NotNull
+	@Column(name = "table_name", length = 100, nullable = false, updatable = false)
+	private String tableName;
+	
+	@NotNull
+	@Column(length = 255, nullable = false, updatable = false)
+	private String identifier;
+	
+	@NotNull
+	@Column(length = 1, nullable = false, updatable = false)
+	private String operation;
+	
+	@NotNull
+	@Column(name = "message_uuid", length = 38, nullable = false, unique = true, updatable = false)
+	private String messageUuid;
+	
+	@Column(name = "request_uuid", length = 38, updatable = false)
+	private String requestUuid;
+	
+	@NotNull
+	@Column(nullable = false, updatable = false)
+	private boolean snapshot;
+	
+	@NotNull
+	@Enumerated(EnumType.STRING)
+	@Column(nullable = false, length = 20)
+	@Access(AccessType.FIELD)
+	private SenderSyncMessageStatus status = SenderSyncMessageStatus.NEW;
+	
+	@Column(name = "date_updated")
+	@Access(AccessType.FIELD)
+	private Date dateUpdated;
```

Comment:
Rename to dateChanged and column name to date_changed

Reply:
Done.